### DISCUSSION TITLE
Inferences are getting better but loss is growing

### DISCUSSION THREAD

**Discussion by AndrzejGajos** [coqui-ai/TTS#2873] [General Q&A]
Hi,

I am finetuning tts_models--pl--mai_female--vits. 

I am following the procedure from the notebook:
https://colab.research.google.com/drive/1xy0qmej_G3skZL2BpY1sBm_k3BTTkf7V?usp=sharing
and everything is working really well, except for loss and models being saved as 'best_model.pth'. When I use checkpoints for tts I am very pleased with how training is progressing, but loss function is telling a different story:

![obraz](https://github.com/coqui-ai/TTS/assets/45202199/9dca0d5c-d7e1-4930-a0b7-5aad84a05e25)

Does anyone have any idea what is happening?

**Comment by erogol** [comment]
In tts loss is not always the best indicator of quality. So if it sounds better with more steps, keep training.
